### DISCUSSION TITLE
Search isn't showing all posts

### DISCUSSION THREAD

**Discussion by Zero3K** [bluesky-social/atproto#1372] [General Discussion]
I am only getting recent pots when I search for something.

**Comment by Zero3K** [comment]
When will it be fixed?

**Comment by bnewbold** [comment]
We are have some work in progress to iterate on our search service, and that should make searching over full history more reliable. We don't have a particular timeline for when that will be done.

**Reply by Zero3K** [reply]
Okay.

**Comment by Zero3K** [comment]
This seems to have been fixed as of recent.

**Reply by pfrazee** [reply]
Yep
